easypost/services/order_service.py

```diff
@@ -28,12 +28,12 @@ def retrieve(self, id: str) -> Order:
 
     def get_next_page(
         self,
-        insurances: Dict[str, Any],
+        orders: Dict[str, Any],
         page_size: int,
         optional_params: Optional[Dict[str, Any]] = None,
     ) -> Dict[str, Any]:
         """Retrieve the next page of the list Order response."""
-        return self._get_next_page_resources(self._model_class, insurances, page_size, optional_params)
+        raise NotImplementedError
```

Comment:
gosh... how did this make it in initially? 😭

Do we want to not implement this or just remove it? It shouldn't have been here at all so I'm less worried about removal here, especially since we don't have an `all` endpoint to pass orders into - and it was mislabeled insurances 😂

Reply:
I'd be down for just ripping it out.

Reply:
Let's rip it out then.

@jchen293 because it shouldn't have ever made it in here because there's no `all` function for orders, you can't pass in the results anyway to get the next page.